python-sdk/src/astro/extractors/extractor.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, List, Optional
```

Comment:
Can you add the following line too plz to take advantage of https://github.com/astronomer/astro-sdk/pull/650:

```python
from __future__ import annotations
```